README.md

```diff
@@ -1,334 +1,27 @@
-
-Given the two cells above, a `POST` request to `/person` produces a HTTP response like the following from the kernel gateway, assuming no errors occur:
-
-```
-HTTP/1.1 200 OK
-Content-Type: application/json
-
-{"id": 123}
-```
-
-See the [setting_response_metadata.ipynb](etc/api_examples/setting_response_metadata.ipynb) notebook for examples of setting response metadata.
-
-### Swagger Spec
-
-The resource `/_api/spec/swagger.json` is automatically generated from the notebook used to define the HTTP API. The response is a simple Swagger spec which can be used with the [Swagger editor](http://editor.swagger.io/#/), a [Swagger ui](https://github.com/swagger-api/swagger-ui), or with any other Swagger-aware tool.
-
-Currently, every response is listed as having a status of `200 OK`.
-
-### Running
-
-The minimum number of arguments needed to run in HTTP mode are `--KernelGatewayApp.api=notebook-http` and `--KernelGatewayApp.seed_uri=some/notebook/file.ipynb`.
-
-If you  development, you can run the kernel gateway in `notebook-http` mode using the Makefile in this repository:
-
-```
-make dev ARGS="--KernelGatewayApp.api='notebook-http' \
---KernelGatewayApp.seed_uri=/srv/kernel_gateway/etc/api_examples/api_intro.ipynb"
-```
-
-With the above Make command, all of the notebooks in `etc/api_examples` are
-mounted into `/srv/kernel_gateway/etc/api_examples/` and can be run in HTTP mode.
-
-The notebook-http mode will honor the `prespawn_count` command line argument. This will start the specified number of kernels and execute the `seed_uri` notebook on each one. Requests will be distributed across the pool of prespawned kernels, providing a minimal layer of scalability. An example which starts a pool of 5 kernels follows:
-
-```
-make dev ARGS="--KernelGatewayApp.api='notebook-http' \
-    --KernelGatewayApp.seed_uri=/srv/kernel_gateway/etc/api_examples/api_intro.ipynb" \
-    --KernelGatewayApp.prespawn_count=5
-```
-
-## Develop
-
-This repository is setup for a Dockerized development environment. On a Mac, do this one-time setup if you don't have a local Docker environment yet.
-
-```
-brew update
-
-# make sure you're on Docker >= 1.7
-brew install docker-machine docker
-docker-machine create -d virtualbox dev
-eval "$(docker-machine env dev)"
-```
-
-Clone this repository in a local directory that docker can volume mount:
-
-```
-# make a directory under ~ to put source
-mkdir -p ~/projects
-cd !$
-
-# clone this repo
-git clone https://github.com/jupyter-incubator/kernel_gateway.git
-```
-
-Run the tests:
-
-```
-make test-python3
-make test-python2
-```
-
-Run the gateway server:
-
-```
-cd kernel_gateway
-make dev
-```
-
-To access the gateway instance:
-
-1. Run `docker-machine ls` and note the IP of the dev machine.
-2. Visit http://THAT_IP:8888/api in your browser. (Note that the route `/api/kernels` is not enabled by default for security. See the `--KernelGatewayApp.list_kernels` parameter above.)
+### Setting up a development system
```

Comment:
Can we retain the Develop section which has the instructions for setting up a dockerized dev environment? We can certainly have other options, but I think having something here to start while alternative approaches are written is better than nothing.